custom_reports/integrations/sharepoint.rst

```diff
@@ -3,6 +3,7 @@ SharePoint
 
 Once SharePoint integration is enabled, additional metrics and dimensions will
 become available in the :ref:`custom-reports-http-api`.
+Sharepoint data isn't available in Real-time API.
```

Comment:
Is this true? Doeas this mean event stream will not have 11 event type  al tall or we will not have additional sharepoint dimensions there?

Reply:
there's no Sharepoint dimensions in Real-time and [Sharepoint is not sold](https://piwikpro.slack.com/archives/C1PH7HZFT/p1707813012073119?thread_ts=1707744657.966529&cid=C1PH7HZFT) anymore, so there should be no new events with sharepoint type